src/pages/trader-tools/common/_utility.js

```diff
@@ -245,6 +250,40 @@ export const resetValidationMargin = (values) => {
     return errors
 }
 
+export const resetSignupAffiliateDetails = (values, is_input_country) => {
+    const errors = {}
+    const firstName_error = validation.firstName(values.firstName)
+    const lastName_error = validation.lastName(values.lastName)
+    const date_error = validation.date(values.date)
+    const country_error = validation.country(values.country, is_input_country)
+    const address_error = validation.address(values.address)
+    const mobileNumber_error = validation.mobileNumber(values.mobileNumber)
+    const password_error = validation.password(values.password)
+
+    if (firstName_error) {
+        errors.firstName = firstName_error
+    }
+    if (lastName_error) {
+        errors.lastName = lastName_error
+    }
+    if (date_error) {
+        errors.date = date_error
+    }
+    if (country_error) {
+        errors.country = country_error
+    }
+    if (address_error) {
+        errors.address = address_error
+    }
+    if (mobileNumber_error) {
+        errors.mobileNumber = mobileNumber_error
+    }
+    if (password_error) {
+        errors.password = password_error
+    }
+    return errors
+}
+
```

Comment:
This portion shouldn't be placed in this file as it may confuse others in the future since this is not related to trader tools.